.github/workflows/php.yml

```diff
@@ -0,0 +1,52 @@
+name: PHP Composer
```

Comment:
"Test coverage" or sth like that, the workflow file could be also called "testing.yml".

Code style checking and static code analyzer is missing from here, it could go to a separate workflow file or here.

I think this PR should be a "Replace Travis CI" which _completely_ replaces Travis and removes it from the repo. Do you agree?

Reply:
Highest-lowest testing is also missing from here, see .travis.yml